app/src/main/java/mozilla/lockbox/presenter/LockedPresenter.kt

```diff
@@ -70,21 +63,22 @@ class LockedPresenter(
                 when (it) {
                     is FingerprintAuthAction.OnSuccess -> unlock()
                     is FingerprintAuthAction.OnError -> unlockFallback()
+                    is FingerprintAuthAction.OnCancel -> dispatcher.dispatch(AutofillAction.Cancel)
                 }
             }
             .addTo(compositeDisposable)
     }
 
-    private fun unlock() {
+    fun unlock() {
         dispatcher.dispatch(DataStoreAction.Unlock)
         dispatcher.dispatch(UnlockingAction(false))
     }
 
     private fun unlockFallback() {
         if (fingerprintStore.isKeyguardDeviceSecure) {
-            dispatcher.dispatch(RouteAction.UnlockFallbackDialog)
+            lockedView.unlockFallback()
```

Comment:
Use: `dispatcher.dispatch(RouteAction.UnlockFallbackDialog)` instead of the `lockedView` method.